cypress/e2e/settings.cy.js

```diff
@@ -10,23 +10,34 @@ describe('Settings page', () => {
 
   });
 
-  it('should provide an ability to update username', () => {
-
+  it('should update bio', () => {
+    cy.get('[data-qa="bio"]').clear().type(faker.lorem.sentence());
+    cy.get('[data-qa="save-profile"]').click();
+    cy.reload();
+    cy.get('[data-qa="bio"]').should('contain', faker.lorem.sentence());
```

Comment:
Using `faker.lorem.sentence()` for both typing and asserting the bio content can lead to flaky tests because `faker` generates different values each time. Consider storing the generated sentence in a variable and using it for both actions.